client/packages/android/app/src/main/java/org/openmsupply/client/DiscoveryConstants.java

```diff
@@ -0,0 +1,21 @@
+package org.openmsupply.client;
```

Comment:
Bot MainActivity.java and NativeApi.java use PORT and hardwareID, thought to make a seperate class so it can be shared